site/content/contribute/team_contributions/_index.md

```diff
@@ -1,14 +1,30 @@
 ---
-title: "Mattermost Contributor Wall of Fame"
+title: "Contributor Wall of Fame"
 section: "contribute"
 ---
 
+# Mattermost Contributor Wall of Fame
+
 Mattermost is an open source Slack-alternative that helps teams stay in sync, with messaging and file sharing across PCs and phones, with archiving and search, and a growing community of integrations, bots and applications.
 
 The project is developed by a global community of contributors and is maintained and supported by a distributed company, [Mattermost Inc.](https://about.mattermost.com/company/), based in Palo Alto, California.
 
-Contributors by GitHub username and (contribution date) are alphabetically listed below. The list is updated monthly with each release, and includes contributions to [mattermost-server](https://github.com/mattermost/mattermost-server) and [mattermost-webapp](https://github.com/mattermost/mattermost-webapp) repositories.
+## Platform Contributors
+
+Platform contributors by GitHub username and (contribution date) are alphabetically listed below. The list is updated regularly (last updated March 20, 2020), and includes contributions to [mattermost-server](https://github.com/mattermost/mattermost-server) and [mattermost-webapp](https://github.com/mattermost/mattermost-webapp) repositories.
```

Comment:
Regarding `last updated` date, how do we determine when the value should be updated?

Reply:
@jfrerich Thanks! Completely agree re: automation.

Ideally we'd update the contributor list (and last updated date) every month as part of our release cycle - however, we sometimes don't have time to do so since it's a manual process.

Would love to automate it though. We had discussed some form of automation in the Community Process channel, wondering if this is something you or someone in the engineering team was open to helping me with?

https://community.mattermost.com/core/pl/nnnxxbgt3frufd5aw4w4zsttee